location/models.py

```diff
@@ -31,22 +32,29 @@ def google_map_url(self):
 
 
 class Apartment(models.Model):
-    suite_num = models.CharField(max_length=30, blank=True, null=True)
+    suite_num = models.CharField(unique=True, max_length=30, blank=True, null=True)
```

Comment:
Just wondering what is the reason to set the `unique=True` here

Reply:
It's because I assumed there is no duplicate apartment with the same suite num.